playbooks/roles/deploy-osh/templates/glance.yaml.j2

```diff
@@ -9,4 +9,16 @@ conf:
     glance_store:
       rbd_store_user: {{ ses_cluster_configuration['glance']['rbd_store_user'] }}
       rbd_store_pool: {{ ses_cluster_configuration['glance']['rbd_store_pool'] }}
-{{ lookup('template','files/common-images-overrides.yml') | from_yaml | to_nice_yaml(indent=2) }}
+{% if developer_mode %}
```

Comment:
Do we still need developer_mode ?
I thought there was no patch to use for this?

Reply:
Right, no patch. I'll remove the developer_mode check